code/modules/mining/ores_coins.dm

```diff
@@ -247,6 +247,17 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 	item_state = "slag"
 	singular_name = "slag chunk"
 
+/obj/item/stack/ore/ice
+	name = "ice crystals"
+	desc = "Used in an electrolyzer to produce hydrogen and oxygen."
+	icon_state = "Ice ore"
+	item_state = "Ice ore"
+	singular_name = "ice chunk"
+	scan_state = "rock_Ice"
```

Comment:
camel + snake case... how evil..

Reply:
Evil, but it's how all ore scan states are written so I'm keeping it consistent